src/styles/globals.css

```diff
@@ -281,6 +285,23 @@ select.short {
   @apply w-min;
 }
 
+button svg,
+.button-md svg {
+  @apply w-5 h-5 mr-2 last:mr-0;
+}
+
+.button-sm svg {
+  @apply w-4 h-4 mr-1.5 last:w-5 last:h-5 last:mr-0;
```

Comment:
Why is the icon larger only if it's the last item?

Reply:
We use 5x5 icons along with the smaller button size when there is no accompanying text (e.g., logs page). It's not so much that it's the last item, but the only item.